Refresh.GameServer/Types/Assets/GameAssetType.cs

```diff
@@ -6,156 +7,654 @@ namespace Refresh.GameServer.Types.Assets;
 public enum GameAssetType
 {
     Unknown = -1,
+    
     /// <summary>
-    /// A level, containing objects and such.
+    /// An LBP texture, wraps over a DDS texture. This is sometimes uploaded by the game in normal operation.
     /// </summary>
     /// <remarks>
-    /// Magic: LVLb
+    /// Magic: "TEX"
     /// </remarks>
-    Level = 0,
+    Texture = 0,
+    
     /// <summary>
-    /// An object, usually for prize bubbles or emitters and such.
+    /// A type of texture generally stored only in the game files, this is only ever uploaded when mods are used.
     /// </summary>
     /// <remarks>
-    /// Magic: PLNb
+    /// Magic: "GTF"
     /// </remarks>
-    Plan = 1,
+    GameDataTexture,
```

Comment:
Are the order changes here accounted for elsewhere? Have you tested migrations?

Reply:
I have tested migrations on my local instance, but i havent done it on a copy of prod yet, i'll do that before marking as ready for review

Reply:
double checked migrations, all works